2_base_and_sensor.ipynb

```diff
@@ -573,6 +1073,19 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>What's next</b>\n",
+    "    <p>By comparing the laser scan variable to the room shape, it is possible to \"localize\" the robot absolute position in the room.</p>\n",
+    "    <p>We will learn how to use high level moving function, based on the localization function installed in HSR.</p>\n",
```

Comment:
First, "moving" is replaced with "movement" and "HSR" is replaced with "the HSR".

Next, we have to somehow deal with the singular and plural references for functions.  Please choose the one that most correctly matches the reality you wish to express:

    "    <p>We will learn how to use a high level movement function, based on the localization function installed in the HSR.</p>\n",

    "    <p>We will learn how to use high level movement functions, based on the localization function installed in the HSR.</p>\n",

    "    <p>We will learn how to use a high level movement function, based on the localization function installed in the HSR.</p>\n",

    "    <p>We will learn how to use high level movement functions, based on the localization functions installed in the HSR.</p>\n",

Reply:
Ah!  It looks like this line (line 1085) has not been replaced, yet.
The previous comment provided 4 sample replacements.
It is only necessary to choose the sentence that most correctly fits with the context.
Any questions, please feel free to ask. :)